examples/requests_adafruit_discord_active_online.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-FileCopyrightText: 2022 DJDevon3 for Adafruit Industries
+# SPDX-License-Identifier: MIT
+# Coded for Circuit Python 8.0
+"""DJDevon3 Adafruit Feather ESP32-S2 Adafruit_Discord_Active_Users_Example"""
+import gc
+import re
+import time
+import ssl
+import json
+import wifi
+import socketpool
+import adafruit_requests
+
+# No user or token required, web scrape.
+
+# Initialize WiFi Pool (There can be only 1 pool & top of script)
+pool = socketpool.SocketPool(wifi.radio)
+
+# Time between API refreshes
+# 900 = 15 mins, 1800 = 30 mins, 3600 = 1 hour
+sleep_time = 900
+
+try:
+    from secrets import secrets
+except ImportError:
+    print("Secrets File Import Error")
+    raise
+
+if sleep_time < 60:
+    sleep_time_conversion = "seconds"
+    sleep_int = sleep_time
+elif 60 <= sleep_time < 3600:
+    sleep_int = sleep_time / 60
+    sleep_time_conversion = "minutes"
+elif 3600 <= sleep_time < 86400:
+    sleep_int = sleep_time / 60 / 60
+    sleep_time_conversion = "hours"
+else:
+    sleep_int = sleep_time / 60 / 60 / 24
+    sleep_time_conversion = "days"
+
+# https://img.shields.io/discord/327254708534116352.svg
+ADA_DISCORD_SVG = "https://img.shields.io/discord/327254708534116352.json"
```

Comment:
Nice find! I didn't know there was JSON data available directly. Since it's not using an svg anymore lets change the variable name to `ADA_DISCORD_JSON` or something that matches the new type instead.